datamodel/initdb.d/03_dcsa_ovs_v1_0.sql

```diff
@@ -4,6 +4,52 @@
 
 
 
+-- OVS Required Reference Data
```

Comment:
Shouldn't all this use DCSA-Information-Model repository? Are you copying data?

Reply:
Will be updated, just some discussion required